migration/migration.go

```diff
@@ -542,3 +543,65 @@ func migrateTrieNodesFromBitsetToTrieKey(target db.Bucket) BucketMigratorDoFunc
 		return txn.Set(append(orgKeyPrefix, keyBuffer.Bytes()...), tempBuf.Bytes())
 	}
 }
+
+func addNetworkBucket(txn db.Transaction, network utils.Network) error { //nolint:gocyclo
```

Comment:
I don't we need to do this. Especially since it wont work for custom networks. We can always populate the `Network` bucket (if empty of course) when we are booting up.

Reply:
> What about for existing db's?

We can assume user will provide the correct network the first time Juno is restarted.

> I think the migration should always work, even after we start supporting custom networks. Am I missing something?

https://github.com/NethermindEth/juno/pull/1461/files#diff-d71e3f116a6ab5e5b759bc2e997dd9c9a77c30eb037e59ffa612a25a18925ad8R548-R571

Genesis hashes are hard coded, how would it work for arbitrary networks?

Reply:
> Genesis hashes are hard coded, how would it work for arbitrary networks?

Oh, just realized , since custom network support will be introduced after this feature we will never run this migration

Reply:
We would on startup, but we can still use the network name, chain ID, or something else to insert in the `Network` bucket.

Reply:
> We can assume user will provide the correct network the first time Juno is restarted and write that to the DB.

We do this in the migration.